package.json

```diff
@@ -11,7 +11,7 @@
     "postinstall": "scripts/cli init_submodule",
     "build": "lerna run build --stream",
     "build:bundle": "lerna run build:bundle --stream",
-    "build:app": "cd test/app && rm -rf node_modules && yarn && yarn build",
+    "build:app": "lerna run pack --stream && cd test/app && rm -rf node_modules && yarn install --no-immutable && yarn build",
```

Comment:
`--no-immutable` is needed as we're packing the dependencies used in `test/app` in the CI, their checksum is likely to changes between build.